src/frontend/src/lib/stores/modal.store.ts

```diff
@@ -32,20 +32,21 @@ export interface Modal<T> {
 		| 'btc-transaction'
 		| 'dapp-details';
 	data?: T;
+	modalId?: symbol;
```

Comment:
I was thinking, it can just be called `id`.

Reply:
yes, better!